test/TensorFlow/deabstraction_finished.swift

```diff
@@ -230,3 +230,20 @@ public func bar() {
 
 // CHECK-LABEL: --- INPUT FUNCTION {{.*}}bar
 // CHECK: graph_op "Const"
+
+// Check that tensorflow convention is propagated to closures from declarations.
+public func testTensorFlowClosures(_ a: Float) -> Tensor<Int32>{
+  let closure: @convention(tensorflow) (Tensor<Float>) -> Tensor<Int32> = {
+    return Tensor<Int32>($0)
+  }
+  let f = Tensor<Float>(a);
+  return closure(f);
+}
+
+// CHECK-LABEL --- TFPartition Accelerator Result: [[NAME:*.*]]
```

Comment:
Actually, this is not a check because it missed a colon.

Reply:
Oops! Will send out a patch.